stubs/noves/NovesTranslate.ts

```diff
@@ -10,9 +11,9 @@ const NOVES_TRANSLATE_CLASSIFIED: NovesClassificationData = {
     to: { name: '', address: '0xa0393A76b132526a70450273CafeceB45eea6dEE' },
     token: {
       address: '',
-      name: 'ETH',
-      symbol: 'ETH',
-      decimals: 18,
+      name: config.chain.currency.name ?? '',
+      symbol: config.chain.currency.symbol ?? '',
+      decimals: config.chain.currency.decimals,
```

Comment:
We never show data from stubs to the user; they are solely used as placeholders for rendering skeletons correctly. Therefore, we would like to keep this data static without any configuration.